contentcuration/contentcuration/frontend/channelList/views/KCard/BaseCard.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+
+  <button>
```

Comment:
[Required] The very outermost element needs to be `<li>`, see "Markup requirements" in https://github.com/learningequality/kolibri-design-system/issues/529, not `<button>` and neither `<div>`. This is important because `KCard` components will be used within `<ul>` and its direct children need to be `<li>` so cards are designed to be list items at this point (we may need to tweak this as we continue working on grids and perhaps there will be an intermediary card grid item component, but for now let's make sure it works from the beginning).

Relatedly, please remove `button` element completely. Semantically the card doesn't have much to do with button functionality and markup needs to reflect that as best possible for assistive technologies.`